src/schema.ts

```diff
@@ -13,7 +14,7 @@ function getSQLVersion(connection: string): SQLVersion {
     } else if (/^mysql:\/\//i.test(connection)) {
         return SQLVersion.MYSQL
     } else {
-        throw new Error(`SQL version unsupported in connection: ${connection}`)
+        return SQLVersion.UNKNOWN
```

Comment:
I think it's still good to keep throwing exception here, Since an unknown SQL protocol can't be handled by schemats